crates/control-interface/src/types.rs

```diff
@@ -224,10 +224,14 @@ pub struct RegistryCredential {
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub username: Option<String>,
     /// The type of the registry (only "oci" is supported at this time")
-    #[serde(rename = "registryType")]
+    #[serde(rename = "registryType", default = "default_registry_type")]
```

Comment:
Nit: this could be `default_value` to avoid the need to define a function

Reply:
i think i tried to use `default_value` but i got an error. i know it works with `clap` but couldn't get it to work with `serde`. can you please paste in the annotation you have in mind

Reply:
Ah, I was thinking of clap, not serde. Disregard this 🙂